src/Compilers/Core/Portable/Operations/IRecursivePatternOperation.cs

```diff
@@ -13,6 +13,25 @@ namespace Microsoft.CodeAnalysis.Operations
     /// </remarks>
     public interface IRecursivePatternOperation : IPatternOperation
```

Comment:
Testing around `HasErrors`, such as when there is no `DeconstructSymbol` but there are patterns.